tulip/transys/transys.py

```diff
@@ -37,6 +37,9 @@
 from tulip.transys.labeled_graphs import (
     LabeledDiGraph, str2singleton, prepend_with)
 from tulip.transys.mathset import PowerSet, MathSet
+from networkx import MultiDiGraph
+from Queue import Queue
```

Comment:
Thanks for the updates. The Python 2 module [`Queue`](https://docs.python.org/2/library/queue.html) has been renamed to [`queue`](https://docs.python.org/3/library/queue.html) in Python 3, which causes some of the tests to fail on Travis CI. A simple solution would be:

```python
try:
    from Queue import Queue
except ImportError:
    from queue import Queue
```